packages/fetch-and-compile/lib/multiple.ts

```diff
@@ -16,8 +17,8 @@ export class MultipleRecognizer implements Recognizer {
 
   constructor(addresses: string[]) {
     this.unrecognizedAddresses = [
-      ...new Set(addresses.map(address => address.toLowerCase()))
-    ]; //remove duplicates (case insensitive) and clone
+      ...new Set(addresses.map(Web3Utils.toChecksumAddress))
```

Comment:
@haltman-at `toChecksumAddress` can throw and interrupt the flow. Of course, if the inputs are valid addresses, there's nothing to worry about.